packages/toolchain/collection.yaml

```diff
@@ -21,22 +21,32 @@ packages:
       autobump.revdeps: "true"
   - name: "luet-makeiso"
     category: "toolchain"
-    version: 0.3.8-15
+    version: 0.3.8-16
     upx: false
     fips: false
     labels:
       github.repo: "luet-makeiso"
       github.owner: "mudler"
       autobump.revdeps: "true"
+    requires:
+    - name: "luet"
+      category: "toolchain"
+      # Would be nice to restrict to the luet version defined within this file
```

Comment:
can use YAML anchors for that: define luet once, and refer it in multiple places, like we do in the cos collection